test/kb_Metrics_server_test.py

```diff
@@ -19,6 +19,7 @@
 from kb_Metrics.kb_MetricsServer import MethodContext
 from kb_Metrics.authclient import KBaseAuth as _KBaseAuth
 from kb_Metrics.metricsdb_controller import MetricsMongoDBController
+from kb_Metrics.metricsDBs import MongoMetricsDBI
```

Comment:
Please run make use you check pep8. 
In general, many of your tests only test base cases and only test return structure's size and key not assert actual value of them. You should redesign your tests so that it covers edge cases and raise statements.

Reply:
I still see pep8 errors.

Reply:
I ran check and made changes on this, except line too long and the lambda suggestions.

Reply:
Those need to be fixed as well.

Reply:
Fixed.